meta/kwoptarg.rb

```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```

Comment:
We just need to eventually add a way of mapping the specs back onto the subject so it doesn't matter and you don't have to worry about fragile things like that.